powerline.el

```diff
@@ -509,16 +509,15 @@ static char * %s[] = {
 ;;;###autoload (autoload 'powerline-buffer-id "powerline")
 (defun powerline-buffer-id (&optional face pad)
   (powerline-raw
-   (format-mode-line
-    (concat " " (propertize
-                 (format-mode-line mode-line-buffer-identification)
-                 'face face
-                 'mouse-face 'mode-line-highlight
-                 'help-echo "Buffer name\n\ mouse-1: Previous buffer\n\ mouse-3: Next buffer"
-                 'local-map (let ((map (make-sparse-keymap)))
-                              (define-key map [mode-line mouse-1] 'mode-line-previous-buffer)
-                              (define-key map [mode-line mouse-3] 'mode-line-next-buffer)
-                              map))))
+   '(" " (:propertize
+          mode-line-buffer-identification
```

Comment:
don't we need at least one call to `format-mode-line`?

Reply:
`powerline-raw` already calls `format-mode-line`, since this function will end up passing it a list, so I don't think it's necessary here.